go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/rancher/rancher
 
-go 1.20
+go 1.21
+
+toolchain go1.21.3
```

Comment:
This shouldn't be here, right? I've seen this myself when updating `go.mod`, but removing it should be fine, right?